bin/ruby-build

```diff
@@ -1194,7 +1198,7 @@ isolated_gem_install() {
 apply_ruby_patch() {
   local patchfile
   case "$1" in
-  ruby-* | jruby-* | rubinius-* | truffleruby-* )
+  ruby-* | jruby-* | mruby-* | picoruby-* | truffleruby-* | truffleruby+graalvm-* )
```

Comment:
This feels like it belongs in a separate PR

Reply:
It's just a cleanup I noticed while working on this and already its own commit, but sure I can split it up.

Reply:
The problem with this change being bundled here, apart from being wholly unrelated, is that it won't get its own changelog entry automatically upon cutting a release with `script/release` (nor when creating a release using the web interface and choosing "Generate release notes"). I believe that all feature changes deserve a changelog entry.

Reply:
In the updated PR it's fully related and necessary.